docs/reference-manual/native-image/JCmd.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: docs
+toc_group: debugging-and-diagnostics
+link_title: JCmd
+permalink: /reference-manual/native-image/debugging-and-diagnostics/JCmd/
+---
+
+# Java Diagnostic Command (jcmd) with Native Image
+
+ Native Image has support for Java Diagnostic Command (`jcmd`). Users can now use the same `jcmd` tool they use to interact with Java applications to interact with Native Image executables. Support for `jcmd` compliments many of the other monitoring features that exist in Native Image such as JDK Flight Recorder, heap dumps, and native memory tracking.  
+
+## Enabling `jcmd` Support
+
+Support for `jcmd` is disabled by default and must be explicitly enabled at build time.
+ 
+To build a native executable with `jcmd` support, use the `--enable-monitoring=jcmd` option.
+```shell
+native-image --enable-monitoring=jcmd YourApplication
+```
+
```

Comment:
Please briefly mention `jvmstat` as well (without `jvmstat` support, `jcmd` won't list the native image process if `jcmd` is executed without any arguments).

Reply:
Good idea! I've added a note about jvmstat